e2e/cypress/support/compliance_export_commands.js

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
I don't think some of these new commands warrant a place in the global commands. Remember that if you add a global command that looks generic and may be accessed by other tests, this may lead to a lot of confusion since those commands look generic whereas they shouldn't be because they cannot be reused.
 
They are at best helper functions that should be moved to
`enterprise/system_console/compliance/helpers.js`

If you want to retain some of these commands as global functions, they should be under `cypress/support/ui` and following naming convention that relates to compliance export like `uiXXXComplianceExportXXX` and with `d.ts` file included
```
cypress/support/ui/compliance_export.js
cypress/support/ui/compliance_export.d.ts
```

I'll let @saturninoabril chime in on this as he might have a better structure for these commands.

Reply:
Thanks @Joseph
Yeah, I agree some commands are very specific to compliance export tests. 
I initially made local functions but I ended up four spec files and all of them are using these functions. 
I think **helper.js** idea is better. 
Let's wait for Saturn response.